src/api/apiKey.jsx

```diff
@@ -0,0 +1 @@
+export const API_KEY = 'key=ea4c76a0ef0d4fc0875f37cdd589291f';
```

Comment:
Surtout pas de clé d'API sur Github !
Il faut utiliser à la place les [variables d'environnement](https://vitejs.dev/guide/env-and-mode.html#env-variables-and-modes).
via un fichier `.env` qui **ne sera pas commit**
et un fichier `.env.sample` qui sera commit avec une valeur d'exemple pour le reste de l'équipe

```.env
# .env
API_KEY=ab0c12d3ef4g5hi6789j01klm234567n
```

```.env
# .env.sample
API_KEY=EXEMPLE_KEY
```